projects/batfish-common-protocol/src/main/java/org/batfish/datamodel/ConfigurationFormat.java

```diff
@@ -8,6 +8,7 @@
   ARISTA("arista"),
   AWS_VPC("aws_vpc"),
   BLADENETWORK("bladenetwork"),
+  CADANT("cadant"),
```

Comment:
Also requires cases in `org.batfish.datamodel.RoutingProtolL:getDefaultAdministrativeCost` and `getSummaryAdministrativeCost`